tests/x11/vlc.pm

```diff
@@ -35,8 +35,15 @@ sub run {
     assert_and_click "vlc-play_button";
     # The video is actually 23 seconds long so give a bit of headroom for
     # startup
-    assert_screen "vlc-done-playing", 90;
-    assert_and_click "close_vlc";
+    assert_screen [qw(vlc-video-info-only vlc-done-playing), 90];
+    if (match_has_tag 'vlc-done-playing') {
+        wait_still_screen;
+        assert_and_click "close_vlc";
+    }
+    elsif (match_has_tag 'vlc-video-info-only') {
```

Comment:
wouldn't you match this needle in cases when the clock just does not immediately jump to a number higher than 0?